tools/releasing/create_binary_release.sh

```diff
@@ -129,8 +129,8 @@ make_python_release() {
   cp ${pyflink_actual_name} "${PYTHON_RELEASE_DIR}/${pyflink_release_name}"
 
   wheel_packages_num=0
-  # py37,py38,py39,py310 for mac and linux (11 wheel packages)
-  EXPECTED_WHEEL_PACKAGES_NUM=11
+  # py38,py39,py310 for mac 10.9, 11.0 and linux (9 wheel packages)
+  EXPECTED_WHEEL_PACKAGES_NUM=9
```

Comment:
Here we've had originally built:
* (py37,py38,py39,py310) X (mac 10.9, 11.0 and linux) = 9
* (py37) X (mac 10.9 and linux) = 2

Now the last one dropped so we should have 9 but I was unable to test it on azure.